ee/ucp/swarm/deploy-to-collection.md

```diff
@@ -57,20 +57,30 @@ services:
 
   wordpress:
     image: wordpress
+    networks:
+      - wp
     ports:
       - 8080:80
     environment:
       WORDPRESS_DB_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
   mysql:
     image: mysql:5.7
+    networks:
+      - wp
     environment:
       MYSQL_ROOT_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
+
+networks:
+  wp:
+    driver: overlay
+    lables:
```

Comment:
s/lables/labels/

Reply:
oops. Copied and then modified my test incorrectly. Fixing